### ISSUE TITLE
There is a conflict with Angular Material v1.1.1

### ISSUE THREAD

**Issue by dunice-potapov** [logbon72/angular-material-datetimepicker#78]
Hello,

I'm trying to install it via Bower and I have an issue
```
ECONFLICT Unable to find suitable version for angular-material
```

I'm using Angular Material v1.1.1.

Could you please provide instructions how to resolve it?

Thanks,
Alexander

**Comment by PowerKiKi** [comment]
This should be solved by https://github.com/logbon72/angular-material-datetimepicker/pull/79

**Comment by hexadecy** [comment]
Also solved by #34
But I prefer to use yarn than bower.
